src/SiteMaster/Core/Auditor/Site/Pages/Queued.php

```diff
@@ -35,6 +35,6 @@ public function getLimit()
 
     public function getOrderBy()
     {
-        return 'ORDER BY priority ASC, start_time ASC';
+        return 'ORDER BY priority ASC, date_created ASC';
```

Comment:
I assume this corrects improper ordering on NULL dates (pages that weren't started). Does this mean that a currently running site scan can be "stalled" by another site that gets crawled faster?